vitest.browser.shared.config.ts

```diff
@@ -32,7 +32,7 @@ export default defineConfig({
       },
     },
     fakeTimers: {
-      toFake: ["setTimeout"],
+      toFake: ["setTimeout", "Date"],
```

Comment:
Can you tell me why this is needed?

Reply:
Several core tests needed the `Date` to also be mockable with virtual time.